Normally this is brought in by libtool.m4 implicitly, but with slibtool
this does not work.

--- a/configure.ac
+++ b/configure.ac
@@ -5,6 +5,8 @@ LT_INIT
 AC_CONFIG_SRCDIR([libass/ass.c])
 AC_CONFIG_HEADERS([config.h])
 
+AC_CANONICAL_HOST
+
 # Checks for programs.
 AC_PROG_CC
 AM_PROG_CC_C_O
